INFO 2025-05-25 16:31:26,148 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:33:22,659 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:38200]
INFO 2025-05-25 16:33:22,670 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:38200]
WARNING 2025-05-25 16:33:25,787 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 34 - string indices must be integers, not 'str'
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 28, in receive
    _ = text_data["message"]
        ~~~~~~~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
WARNING 2025-05-25 16:33:38,407 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 34 - string indices must be integers, not 'str'
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 28, in receive
    _ = text_data["message"]
        ~~~~~~~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
WARNING 2025-05-25 16:34:10,272 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 34 - string indices must be integers, not 'str'
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 28, in receive
    _ = text_data["message"]
        ~~~~~~~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
WARNING 2025-05-25 16:34:13,061 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 34 - string indices must be integers, not 'str'
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 28, in receive
    _ = text_data["message"]
        ~~~~~~~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
INFO 2025-05-25 16:34:46,401 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:34:47,095 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:35:01,505 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:35:02,060 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:35:11,707 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:57140]
INFO 2025-05-25 16:35:11,714 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:57140]
CRITICAL 2025-05-25 16:35:12,451 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 28 - <class 'str'>
WARNING 2025-05-25 16:35:12,452 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 36 - string indices must be integers, not 'str'
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 29, in receive
    _ = text_data["message"]
        ~~~~~~~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
INFO 2025-05-25 16:35:31,771 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:35:32,322 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:35:33,101 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:41358]
INFO 2025-05-25 16:35:33,109 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:41358]
ERROR 2025-05-25 16:35:33,145 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 28 - <class 'str'>
WARNING 2025-05-25 16:35:33,145 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 36 - string indices must be integers, not 'str'
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 29, in receive
    _ = text_data["message"]
        ~~~~~~~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
ERROR 2025-05-25 16:35:35,905 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 28 - <class 'str'>
WARNING 2025-05-25 16:35:35,906 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 36 - string indices must be integers, not 'str'
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 29, in receive
    _ = text_data["message"]
        ~~~~~~~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
ERROR 2025-05-25 16:35:43,989 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 28 - <class 'str'>
WARNING 2025-05-25 16:35:43,989 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 36 - string indices must be integers, not 'str'
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 29, in receive
    _ = text_data["message"]
        ~~~~~~~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
ERROR 2025-05-25 16:35:45,148 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 28 - <class 'str'>
WARNING 2025-05-25 16:35:45,148 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 36 - string indices must be integers, not 'str'
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 29, in receive
    _ = text_data["message"]
        ~~~~~~~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
ERROR 2025-05-25 16:35:51,180 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 28 - <class 'str'>
WARNING 2025-05-25 16:35:51,181 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 36 - string indices must be integers, not 'str'
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 29, in receive
    _ = text_data["message"]
        ~~~~~~~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
INFO 2025-05-25 16:36:09,028 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:36:09,633 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:36:31,145 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:36:31,652 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:37:11,360 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:37:11,974 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:37:18,425 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:50778]
INFO 2025-05-25 16:37:18,432 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:50778]
WARNING 2025-05-25 16:37:20,980 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 37 - Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 28, in receive
    json_data = json.loads(text_data)
  File "/usr/lib/python3.13/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^
  File "/usr/lib/python3.13/json/decoder.py", line 345, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/json/decoder.py", line 363, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
WARNING 2025-05-25 16:37:43,600 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 37 - Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 28, in receive
    json_data = json.loads(text_data)
  File "/usr/lib/python3.13/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^
  File "/usr/lib/python3.13/json/decoder.py", line 345, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/json/decoder.py", line 363, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
WARNING 2025-05-25 16:38:30,556 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 37 - Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 28, in receive
    json_data = json.loads(text_data)
  File "/usr/lib/python3.13/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^
  File "/usr/lib/python3.13/json/decoder.py", line 345, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/json/decoder.py", line 363, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
INFO 2025-05-25 16:39:03,888 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:39:04,413 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:39:05,728 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:56802]
INFO 2025-05-25 16:39:05,733 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:56802]
WARNING 2025-05-25 16:39:07,025 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 38 - Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 29, in receive
    json_data = json.loads(text_data)
  File "/usr/lib/python3.13/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^
  File "/usr/lib/python3.13/json/decoder.py", line 345, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/json/decoder.py", line 363, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
WARNING 2025-05-25 16:39:36,432 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 38 - Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 29, in receive
    json_data = json.loads(text_data)
  File "/usr/lib/python3.13/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^
  File "/usr/lib/python3.13/json/decoder.py", line 345, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/json/decoder.py", line 363, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
WARNING 2025-05-25 16:39:38,039 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 38 - Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 29, in receive
    json_data = json.loads(text_data)
  File "/usr/lib/python3.13/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^
  File "/usr/lib/python3.13/json/decoder.py", line 345, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/json/decoder.py", line 363, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
WARNING 2025-05-25 16:39:39,774 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 38 - Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 29, in receive
    json_data = json.loads(text_data)
  File "/usr/lib/python3.13/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^
  File "/usr/lib/python3.13/json/decoder.py", line 345, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/json/decoder.py", line 363, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
INFO 2025-05-25 16:39:47,302 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:39:47,936 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:39:49,784 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:50024]
INFO 2025-05-25 16:39:49,793 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:50024]
WARNING 2025-05-25 16:39:50,582 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 38 - Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 29, in receive
    json_data = json.loads(text_data)
  File "/usr/lib/python3.13/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^
  File "/usr/lib/python3.13/json/decoder.py", line 345, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/json/decoder.py", line 363, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
WARNING 2025-05-25 16:39:52,346 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 38 - Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 29, in receive
    json_data = json.loads(text_data)
  File "/usr/lib/python3.13/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^
  File "/usr/lib/python3.13/json/decoder.py", line 345, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/json/decoder.py", line 363, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
INFO 2025-05-25 16:40:27,877 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:40:31,223 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:46488]
INFO 2025-05-25 16:40:31,230 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:46488]
WARNING 2025-05-25 16:40:31,794 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 38 - Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 29, in receive
    json_data = json.loads(text_data)
  File "/usr/lib/python3.13/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^
  File "/usr/lib/python3.13/json/decoder.py", line 345, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/json/decoder.py", line 363, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
WARNING 2025-05-25 16:40:51,341 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 38 - string indices must be integers, not 'str'
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 31, in receive
    _ = text_data["message"]
        ~~~~~~~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
WARNING 2025-05-25 16:40:55,220 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 38 - string indices must be integers, not 'str'
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 31, in receive
    _ = text_data["message"]
        ~~~~~~~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
WARNING 2025-05-25 16:41:05,353 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 38 - string indices must be integers, not 'str'
Traceback (most recent call last):
  File "/home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py", line 31, in receive
    _ = text_data["message"]
        ~~~~~~~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
INFO 2025-05-25 16:41:18,923 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:41:19,468 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:41:21,386 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:59830]
INFO 2025-05-25 16:41:21,391 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:59830]
INFO 2025-05-25 16:41:21,938 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 199 - WebSocket DISCONNECT /ws/chat/ [127.0.0.1:59830]
INFO 2025-05-25 16:41:22,735 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:59846]
INFO 2025-05-25 16:41:22,737 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:59846]
CRITICAL 2025-05-25 16:41:22,758 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 33 - {
    "message":"ss"
}
INFO 2025-05-25 16:41:23,614 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 199 - WebSocket DISCONNECT /ws/chat/ [127.0.0.1:59846]
INFO 2025-05-25 16:41:30,731 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:36962]
INFO 2025-05-25 16:41:30,733 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:36962]
CRITICAL 2025-05-25 16:41:31,625 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 33 - {
    "message":"ss"
}
INFO 2025-05-25 16:41:32,627 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 199 - WebSocket DISCONNECT /ws/chat/ [127.0.0.1:36962]
INFO 2025-05-25 16:42:15,511 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:42:16,148 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:42:17,962 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:42964]
INFO 2025-05-25 16:42:17,968 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:42964]
CRITICAL 2025-05-25 16:42:18,675 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 33 - {
    "message":"ss"
}
INFO 2025-05-25 16:42:35,756 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:42:36,341 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:42:37,677 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:57462]
INFO 2025-05-25 16:42:37,684 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:57462]
CRITICAL 2025-05-25 16:42:37,705 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 33 - {
    "message":"ss"
}
CRITICAL 2025-05-25 16:42:38,334 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 33 - {
    "message":"ss"
}
CRITICAL 2025-05-25 16:42:39,613 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 33 - {
    "message":"ss"
}
INFO 2025-05-25 16:44:43,413 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:44:44,006 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:45:55,310 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:45:55,838 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:45:57,612 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:55906]
INFO 2025-05-25 16:45:57,619 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:55906]
INFO 2025-05-25 16:46:23,523 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:46:24,078 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:46:35,467 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:46:36,104 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:46:36,810 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:35844]
INFO 2025-05-25 16:46:36,816 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:35844]
INFO 2025-05-25 16:46:51,565 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:46:52,236 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:47:02,615 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:47:03,262 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:47:04,122 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:45438]
INFO 2025-05-25 16:47:04,129 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:45438]
INFO 2025-05-25 16:47:18,705 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:47:19,231 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:47:20,900 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:42286]
INFO 2025-05-25 16:47:20,908 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:42286]
INFO 2025-05-25 16:47:39,839 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:47:40,450 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:47:41,868 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:60168]
INFO 2025-05-25 16:47:41,876 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:60168]
CRITICAL 2025-05-25 16:47:42,325 /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py consumers 30 - sSSSSSSSSSs ss
INFO 2025-05-25 16:48:11,185 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:48:11,876 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:48:12,145 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:34196]
INFO 2025-05-25 16:48:12,150 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:34196]
INFO 2025-05-25 16:48:24,297 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:48:24,895 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
INFO 2025-05-25 16:48:25,281 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 201 - WebSocket HANDSHAKING /ws/chat/ [127.0.0.1:56090]
INFO 2025-05-25 16:48:25,286 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 197 - WebSocket CONNECT /ws/chat/ [127.0.0.1:56090]
INFO 2025-05-25 16:48:26,045 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/daphne/management/commands/runserver.py runserver 199 - WebSocket DISCONNECT /ws/chat/ [127.0.0.1:56090]
INFO 2025-05-25 16:48:31,211 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 266 - /home/ali/Desktop/Django/ChatBox/src/private_module/consumers.py changed, reloading.
INFO 2025-05-25 16:48:31,929 /home/ali/Desktop/Django/ChatBox/.venv/lib/python3.13/site-packages/django/utils/autoreload.py autoreload 668 - Watching for file changes with StatReloader
